docs/common/installation.md

```diff
@@ -71,7 +71,12 @@ at GitHub:
     git clone https://github.com/RDycore/RDycore.git
     ```
 
-This places an `RDycore` folder into your current path.
+This places an `RDycore` folder into your current path. Then we to update the submodules.
```

Comment:
How about "Then update the submodules:" instead of "Then we to update the submodules"?

Reply:
(Thanks for adding this!)

Reply:
Fixed